sftp-server.c

```diff
@@ -1743,7 +1783,18 @@ sftp_server_main(int argc, char **argv, struct passwd *user_pw)
 	}
 
 	log_init(__progname, log_level, log_facility, log_stderr);
-
+#ifdef WINDOWS
+	/*
+	 * SSHD process running in SYSTEM will write the logs in sftp-server.log.
```

Comment:
sftp-server forwards the log messages to SSHD System process. SSHD system process logs the messages to either ETW / sftp-server.log. This allows us to log the messages of both non-admin user and admin users.